vllm/lora/models.py

```diff
@@ -215,9 +216,22 @@ def from_local_checkpoint(
         new_embeddings_bin_file_path = os.path.join(lora_dir,
                                                     "new_embeddings.bin")
 
+        tensorizer_config = kwargs.get("tensorizer_config")
+
         unexpected_modules: List[Union[list[str], str]]
-        if os.path.isfile(lora_tensor_path):
-            tensors: Dict[str, torch.Tensor] = {}
+        tensors: Dict[str, torch.Tensor] = {}
+        if tensorizer_config:
+            from tensorizer import TensorDeserializer
+            lora_tensor_path = os.path.join(tensorizer_config.tensorizer_dir,
+                                            "adapter_model.tensors")
+            tensorizer_args = tensorizer_config._construct_tensorizer_args()
+            tensors = TensorDeserializer(
+                lora_tensor_path,
+                dtype=tensorizer_config.dtype,
+                device=f'cuda:{torch.cuda.current_device()}',
+                **tensorizer_args.deserializer_params)
```

Comment:
Tensorizer's default is to use `torch.device("cuda")` if CUDA is available (and `torch.device("cpu")` otherwise), which will generally correspond to the same device as `cuda:{torch.cuda.current_device()}` unless you expect the contextual current device to change for some reason in the middle of reading the tensors. In other words, you shouldn't have to specify this here.
```suggestion
            tensors = TensorDeserializer(
                lora_tensor_path,
                dtype=tensorizer_config.dtype,
                **tensorizer_args.deserializer_params)
```